MeetingBar/Helpers.swift

```diff
@@ -189,6 +190,9 @@ func openEvent(_ event: EKEvent) {
             }
         }
         openMeetingURL(meeting.service, meeting.url, nil)
+    } else if eventUrl != "" {
+        // NSLog("opening \(eventUrl) in default browser")
+        event.url?.openInDefaultBrowser()
```

Comment:
```suggestion
    } else if let eventUrl = event.url {
        eventUrl.openInDefaultBrowser()
```

Looks like you can do this all in one string.
Please check this!

Reply:
Neat trick! @leits 
Looks like you're right of course. I committed that fix, tested & works.